_posts/2016/2016-01-16-annual-update.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+category : blog
+tags : [blog]
+title : SanDiego.js Community in the new year
+topic : SanDiego.js Community in the new year
+details : A retrospective and prospective on our community and our goals
+---
+{% include JB/setup %} 
+
+### Opening thoughts :)
+
+The organizers for the SanDiego.js Community did something different at the beginning of this year. We got together for a kick-off meeting to talk about what we did in 2015, and what our goals are for 2016. In hindsight, I don't now why we haven't done this before.
+
+We have a lot of great changes in store for this year and always welcome feedback. So please take the time to look over our ideas, and share your thoughts in the comments!
+
+### 2015 introduced a lot of change to the SanDiego.js Community
+
+* We built a [solid group of organizers](http://sandiegojs.org/about.html) that are dedicated to the community and nurturing the technology scene in San Diego
+* We said goodbye to one of the original organizers of our community.
```

Comment:
Should we name Kevin here?

We could parenthetically note that we said goodbye to Jarrod in 2014 (just an idea).